src/apic-extension/azext_apic_extension/aaz/latest/apic/api/_create.py

```diff
@@ -399,7 +402,7 @@ def _build_schema_on_200_201(cls):
                 flags={"read_only": True},
             )
             _schema_on_200_201.properties = AAZObjectType(
-                flags={"client_flatten": True},
+                flags={"required": True, "client_flatten": True},
```

Comment:
Confirmed with Azure CLI team, the change is expected and does not impact users.

Background:
typespec has a bug that treats some required parameter as optional. I patch the spec in the way suggested by Azure CLI team and causes this change. CLI users will not be aware of this change.